test/client/http_service.test.js

```diff
@@ -76,23 +76,23 @@ describe('Executing http service functions', () => {
 
       describe('Submitting an invalid request', () => {
         it('should fail when an invalid url value is supplied', async() => {
-          expect(service.get({ url: 'invalidurl' })).rejects.toMatchObject({
+          await expect(service.get({ url: 'invalidurl' })).rejects.toMatchObject({
             name: 'StitchError',
             code: 'FunctionExecutionError',
             response: { status: 400 }
           });
         });
 
         it('should fail when an invalid url datatype is supplied', async() => {
-          expect(service.get({ url: 10281995 })).rejects.toMatchObject({
+          await expect(service.get({ url: 10281995 })).rejects.toMatchObject({
```

Comment:
[nit] I tend to place await directly in front of the promise declaration, as opposed to the outermost declaration.

Reply:
This seemed to be the style the jest docs use